generators/boilerplate/templates/index.js

```diff
@@ -1,3 +1,4 @@
-'use strict';
+<% if (babel) { %>export default {};
```

Comment:
Babel transpiles to [common js modules by default](http://babeljs.io/repl/#?experimental=true&evaluate=true&loose=true&spec=false&playground=false&code=export%20default%20%7B%7D%3B)